packages/base/src/commands.ts

```diff
@@ -200,6 +214,70 @@ namespace Private {
     };
   }
 
+  export function createVectorTileLayer(
+    tracker: WidgetTracker<JupyterGISWidget>
+  ) {
+    return async (args: any) => {
+      const current = tracker.currentWidget;
+
+      if (!current) {
+        return;
+      }
+
+      const form = {
+        title: 'Vector Tile Layer parameters',
+        default: (model: IJupyterGISModel) => {
+          return {
+            name: 'Vector Tile Source',
+            maxZoom: 24,
+            minZoom: 0
+          };
+        }
+      };
+
+      const dialog = new FormDialog({
```

Comment:
There is no parameters about the vector layer (color, opacity...)

We can fix it with the suggestion above, about refactoring the `GeoJSONLayerDialog`